index.test-d.ts

```diff
@@ -34,15 +34,15 @@ expectType<string>(ansiEscapes.clearScreen);
 expectType<string>(ansiEscapes.clearTerminal);
 expectType<string>(ansiEscapes.beep);
 expectType<string>(ansiEscapes.link('foo', 'https://foo.bar'));
-expectType<string>(ansiEscapes.image(Buffer.from(1)));
-expectType<string>(ansiEscapes.image(Buffer.from(1), {width: 1}));
-expectType<string>(ansiEscapes.image(Buffer.from(1), {width: '1px'}));
-expectType<string>(ansiEscapes.image(Buffer.from(1), {width: 'auto'}));
-expectType<string>(ansiEscapes.image(Buffer.from(1), {height: 1}));
-expectType<string>(ansiEscapes.image(Buffer.from(1), {height: '1px'}));
-expectType<string>(ansiEscapes.image(Buffer.from(1), {height: 'auto'}));
+expectType<string>(ansiEscapes.image(Buffer.from('1')));
+expectType<string>(ansiEscapes.image(Buffer.from('1'), {width: 1}));
+expectType<string>(ansiEscapes.image(Buffer.from('1'), {width: '1px'}));
+expectType<string>(ansiEscapes.image(Buffer.from('1'), {width: 'auto'}));
+expectType<string>(ansiEscapes.image(Buffer.from('1'), {height: 1}));
+expectType<string>(ansiEscapes.image(Buffer.from('1'), {height: '1px'}));
+expectType<string>(ansiEscapes.image(Buffer.from('1'), {height: 'auto'}));
```

Comment:
These are not equivalent...

Reply:
I thought `tsd` only tested type definitions, and further I thought that `Buffer.from(1)` was just used to obtain an arbitrary `Buffer` instance.  So here we're obtaining an arbitrary buffer instance again. Did I err here?

Reply:
Actually looking at this again, you're right. Idk what passing scalar `1` was originally intending to do but it is indeed equivalent looking at the docs.